packages/client/tests/functional/_utils/idForProvider.ts

```diff
@@ -1,6 +1,26 @@
-const sqlId = 'String @id @default(uuid())'
-const mongoDbId = 'String @id @default(auto()) @map("_id") @db.ObjectId'
+export interface IOptions {
```

Comment:
just `Options` is fine, we don't have a convention to prefix interfaces with `I`